src/controllers/global.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  getAllDonators: (req, res) => {
+    res.send('All donators');
+  },
+  getAllItems: (req, res) => {
+    res.send('All items');
+  },
+};
```

Comment:
Now that you have kept only one controller for `getAllItems`, I think it might make sense to move `getAllItems` to the items controller and `getAllDonators` to the users controller.

You can handle auth-based conditional logic such as showing all items to unauthenticated users and specific items to authenticated users within the controller itself.

Reply:
OK, thank you. Noted.